tests/lib/ocpp/v201/mocks/evse_manager_fake.hpp

```diff
@@ -56,7 +56,8 @@ class EvseManagerFake : public EvseManagerInterface {
 
     void open_transaction(int evse_id, const std::string& transaction_id) {
         auto& transaction = this->transactions.at(evse_id - 1);
-        transaction = std::make_unique<EnhancedTransaction>();
+        // Really dirty way of fixing unit test. Make proper interface for transactions to solve this
```

Comment:
:scream:

Reply:
Found another dirty but less dirty way. I have some ideas on how to improve the transaction class that will solve this but that is too much for this PR.